bitnami/cert-manager/Chart.yaml

```diff
@@ -35,4 +35,4 @@ maintainers:
 name: cert-manager
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/cert-manager
-version: 0.11.13
+version: 0.11.14
```

Comment:
Could you please update another patch version? (0.11.15)
Version 0.11.14 has already been released https://github.com/bitnami/charts/pull/18648

Reply:
Updated.